phpstorm/setting_up_environment.md

```diff
@@ -2,42 +2,30 @@
 title: Setting-up the Environment for PhpStorm Plugin Development
 ---
 
-## General information
-Follow steps that are described at [Getting Started with Plugin Development](/basics/getting_started.md).
+### General information
+Please review the information in the [Getting Started with Plugin Development](/basics/getting_started.md)
+section of this guide.
 
 There are two ways to develop plugins for PhpStorm:
+1. Use IntelliJ IDEA Ultimate as the IDE with the PHP plugin installed.
+2. Use PhpStorm as the IDE.
 
-1. Use IntelliJ IDEA Ultimate with the PHP plugin installed.
-2. Use PhpStorm as a targeted IDE in a first place.
+The choice affects how you will configure an SDK for developing a plugin. Using IntelliJ IDEA Ultimate as the IDE requires basing the
+SDK for your project on your installed version of IntelliJ IDEA. Using PhpStorm as the IDE requires basing the SDK for
+your project on your installed version of PhpStorm. Choosing an installed IDE as the basis for a project is described on 
+the [Setting Up a Development Environment](/basics/getting_started/setting_up_environment.md) page.
 
-The choice affects how you will configure SDK for the plugin. In the first case you need to specify current installation of IntelliJ IDEA as SDK and in the second case you need to specify current installation of PhpStorm. This step is described in [Getting Started with Plugin Development](/basics/getting_started.md).
+> IntelliJ IDEA Community Edition does not support the PHP plugin. The IDE for developing a PhpStorm plugin can be IntelliJ IDEA Community Edition, but the IDE used for testing must be IntelliJ IDEA Ultimate.
 
-## How to use OpenAPI library
+### How to use the PhpStorm OpenAPI Library for IntelliJ Plugin Devlopment
```

Comment:
I don't think we should mention "IntelliJ Plugin Development" here - the article covers plugins for PhpStorm, not for IntelliJ.